lib/vaahextendflutter/env.dart

```diff
@@ -281,3 +294,18 @@ class PusherConfig {
     );
   }
 }
+
+class HiveConfig {
+  final String directoryName;
+  final Future<Directory> _appDocDirectory = getApplicationDocumentsDirectory();
+
+  HiveConfig({this.directoryName = 'dir'});
+
+  Future<void> init() async {
+    await _appDocDirectory;
+    Directory dir = await _appDocDirectory;
+    await Directory('${dir.path}/dir').create(recursive: true).then((Directory directory) async {
+      Hive.init(directory.path);
+    });
+  }
+}
```

Comment:
config are supposed to hold config values only & no methods, having init method here is worst practice. please check sentry config & implementation.